packages/core/src/services/ai/index.ts

```diff
@@ -173,50 +159,30 @@ export async function ai({
       experimental_telemetry: {
         isEnabled: true,
       },
-    }
-
-    if (schema && output) {
-      const result = streamObject({
-        ...commonOptions,
-        schema: jsonSchema(schema),
-        // output is valid but depending on the type of schema
-        // there might be a mismatch (e.g you pass an object schema but the
-        // output is "array"). Not really an issue we need to defend atm.
-        output: output as any,
-      })
-
-      return Result.ok({
-        fullStream: result.fullStream,
-        object: result.object,
-        usage: result.usage,
-        providerMetadata: result.providerMetadata,
-        type: 'object',
-        providerName: providerType,
-      })
-    }
```

Comment:
No more `streamObject`. Now `streamText` supports `experimental_output`. This was causing a bug where a prompt with schema + tools the tools were never called 🐛